tests/x11regressions/firefox/firefox_passwd.pm

```diff
@@ -72,6 +72,7 @@ sub run {
     send_key "alt-shift-p";    #"Saved Passwords..."
     send_key "alt-shift-p";    #"Show Passwords"
     type_string $masterpw. "\n";
+    sleep 5;                   # attempt to fix passwd issue
```

Comment:
[Here](https://openqa.suse.de/tests/1266925#step/firefox_passwd/27) I see that unexpected pop breaks on line 

```assert_and_click('firefox-passwd-security');```

So sleep won;t help there. And is it possible to replace sleep with screen assertion? We can see a lot of issues with code which presses keys without verification of steps as sometimes are pressed too quickly.